ModernFlyouts/Controls/SessionControl.xaml.cs

```diff
@@ -460,7 +461,7 @@ private async void UpdateSessionInfo(GlobalSystemMediaTransportControlsSession s
 
                 UpdatePlaybackInfo(session);
 
-                await SetThumbnailAsync(mediaInfo.Thumbnail, playback.PlaybackType);
+                await SetThumbnailAsync(mediaInfo.Thumbnail, session.GetPlaybackInfo().PlaybackType);
```

Comment:
Revert this one too.

The `GetPlaybackInfo()` call is expensive. Calling it multiple times will result in increase in memory usage.